packages/mds-policy-service/@types/intents/index.ts

```diff
@@ -2,61 +2,120 @@ import type { BaseRule } from '../models'
 import type { BaseIntentPolicyUserFields, INTENT_TYPE, PartialMicroMobilityStatesMap } from './common'
 import { COMMON_INTENT_RULE_DEFAULTS } from './common'
 
-export type IntentRuleDefaults = Omit<BaseRule<PartialMicroMobilityStatesMap>, 'rule_id'>
+export type IntentRuleDefaults = Omit<BaseRule<PartialMicroMobilityStatesMap>, 'rule_id' | 'modality'> & {
+  modality: 'micromobility'
+}
 
 export const NO_PARKING_INTENT_RULE_CONSTANTS: IntentRuleDefaults = {
+  ...COMMON_INTENT_RULE_DEFAULTS,
+  modality: 'micromobility',
   name: 'No Parking Zone',
   maximum: 0,
-  states: { available: [], non_operational: [], reserved: [], unknown: [] },
-  ...COMMON_INTENT_RULE_DEFAULTS
+  states: { available: [], non_operational: [], reserved: [], unknown: [] }
 }
+
 export const PERMITTED_VEHICLE_COUNT_INTENT_RULE_CONSTANTS: IntentRuleDefaults = {
+  ...COMMON_INTENT_RULE_DEFAULTS,
+  modality: 'micromobility',
   maximum: 0,
   minimum: 0,
   name: 'Permitted Vehicle Count',
-  states: {},
-  ...COMMON_INTENT_RULE_DEFAULTS
+  states: {}
+}
+
+export const PARKING_TIME_LIMIT_INTENT_RULE_CONSTANTS: IntentRuleDefaults = {
+  ...COMMON_INTENT_RULE_DEFAULTS,
+  modality: 'micromobility',
+  rule_units: 'minutes',
+  rule_type: 'time',
+  maximum: 0,
+  minimum: 0,
+  name: 'Parking Time Limit',
+  states: { available: [], non_operational: [], reserved: [] }
+}
+
+export const REQUIRED_PARKING_INTENT_RULE_CONSTANTS: IntentRuleDefaults = {
+  ...COMMON_INTENT_RULE_DEFAULTS,
+  modality: 'micromobility',
+  maximum: 0,
+  minimum: 0,
```

Comment:
Should minimum be specified here? Seems like the maximum ought to be sufficient!

Reply:
It is, but it just feels weird to me to have one end of a numeric window be `null` or `undefined`.